manifests/profiles/default.yaml

```diff
@@ -6,166 +6,27 @@ spec:
   hub: gcr.io/istio-testing
   tag: latest
 
-  # You may override parts of meshconfig by uncommenting the following lines.
-  meshConfig:
-    defaultConfig:
-      proxyMetadata: {}
-    enablePrometheusMerge: true
-    # Opt-out of global http2 upgrades.
-    # Destination rule is used to opt-in.
-    # h2_upgrade_policy: DO_NOT_UPGRADE
-
-  # Traffic management feature
+  # Turn on default components: base, pilot, and ingress gateway
   components:
     base:
       enabled: true
     pilot:
       enabled: true
-
     # Istio Gateway feature
     ingressGateways:
-    - name: istio-ingressgateway
-      enabled: true
+      - name: istio-ingressgateway
```

Comment:
typo?  I think most of the other examples don't indent the `-`

Reply:
IDE likes to add them :-(

Reply:
done